webStorm.config.js

```diff
@@ -0,0 +1,5 @@
+System.config({
```

Comment:
This file should be added to gitignore, right?

Reply:
ye that's fair